src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -369,27 +339,12 @@ private static FlutterSdk getFlutterFromDartSdkLibrary(Library lib, VirtualFile
   }
 
   public enum Command {
-
-    CREATE("Flutter create", "create") {
-      @Override
-      void onStart(@Nullable Module module, @Nullable VirtualFile workingDir, @NotNull String... args) {
-        // Enable Dart.
-        if (module != null) {
-          ApplicationManager.getApplication()
-            .invokeLater(() -> ApplicationManager.getApplication().runWriteAction(() -> DartPlugin.enableDartSdk(module)));
-        }
-      }
```

Comment:
Ah, right. My only reservation is that if there are any IML format changes running it from the IDE (possibly redundantly) ensures that things work but it sounds like we're committed to keeping this up to date so NP.

Thanks!